### ISSUE TITLE
Feature Request: Search bar on all pages

### ISSUE THREAD

**Issue by BaconCatBug** [EVE-Tools/element43#53]
I often need to make multiple searches. When I search for an Item, it takes me to a new page, with no way to search for another item without clicking back or opening a new Element-43 tab. It would be nice to have a search bar at the top of each items page that opens in a new tab so we could do multiple searches quickly.

**Comment by zweizeichen** [comment]
Thank you for your suggestion! I think this would be a useful addition to E43. Another search form opening new windows could be put into the toolbar component as you suggested.

note: check-off item in #39 once this one is done

**Comment by zweizeichen** [comment]
There is a search bar on all pages now, the main search bar is autofocused on page load and you can right click result's names to open them in a new tab now. `ctrl+click` does not work properly as it also selects an item from the list at the same time. This is due to the way the input control works and I could not find an elegant solution which does not break any other behaviour. 

If you find a better solution, feel free to make a PR over at [vue43](https://github.com/eve-tools/vue43).
